src/pages/gown.js

```diff
@@ -0,0 +1,375 @@
+`;
+
+const DetailsFeatures = styled.p`
+display: none;
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    display: inline-block;
+    width: 54%;
+    margin-left: 8%;
+    margin-bottom: 4%;
+    font-family: Raleway;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 14px;
+  }
+`;
+
+const Description = styled.p`
+display: none;
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    display: inline-block;
+    width: 54%;
+    margin-left: 8%;
+    font-family: Raleway;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+  }
+`;
+
+const DetailsAndFeaturesContainer = styled.div`
+  display: flex;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  margin: 0 0 36px 8.8%;
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    border: 1px solid red;
+    width: 40%;
+    display: none;
+  }
+`;
+
+const Details = styled.p`
+@media (min-width: ${props => props.theme.breakpoints.tablet}) {
+}`;
+
+const DesktopButtonContainer = styled.p`
+  display: none;
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    display: flex;
+    margin-left: 8%;
+    margin-top: 24%;
+}
+`;
+
+const MobileButtonContainer = styled.p`
+  display: flex;
+  justify-content: center;
+  border-bottom: 48px solid ${props => props.theme.colors.primaryPink};
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+  display: none;
+}
+`;
+
+const DifferentAnglesContainer = styled.div`
+display: none;
+@media (min-width: ${props => props.theme.breakpoints.tablet}) {
+  display: flex;
+  margin-top: 30px;
+  margin-left: 10%;
+  border-bottom: 200px solid ${props => props.theme.colors.backgroundGray};
+}
+`
+
+const DifferentAngles = styled.img`
+display: none;
+@media (min-width: ${props => props.theme.breakpoints.tablet}) {
+  display: inline-block;
+  width: 24%;
+  height: 300px;
+  margin: 0 26px 0 0;
+}
+`
+
+const GownsPage = () => (
+  <Layout>
+    <NavBar />
+    <LogoContainer>
+      <Logo notag />
+    </LogoContainer>
+    <NavMenuDesktop />
+    <BodyContainer>
+    <SearchMenuContainer>
+      <TitleContainer>
+        <LeftArrowContainer>
+          <Icons name={["fas", "arrow-left"]} size="1x" />
+        </LeftArrowContainer>
+        <CollectionTitle>{collectionTitle[0]}</CollectionTitle>
+        <RightArrowContainer>
+          <Icons name={["fas", "arrow-right"]} size="1x" />
+        </RightArrowContainer>
+      </TitleContainer>
+      <DesktopCollectionTitleContainer>
+        <CollectionTitle>Fall 2020</CollectionTitle>
+      </DesktopCollectionTitleContainer>
+    </SearchMenuContainer>
+    <ImageAndDetailsContainer>
+    <GownImage src={cardData[0].image} />
+    <DesktopFlexColumn>
+    <Name>{cardData[0].mobileLabel}</Name>
+    <Brief>Stuff</Brief>
+    <DetailsFeatures>Details and Features</DetailsFeatures>
+    <Description>Description with a whole bunch of stuff describing the gown</Description>
```

Comment:
Try adding in some info about a dress to show what it would actually look like.

Reply:
added text from Figma mockup